src/core/abr/representation_estimator.ts

```diff
@@ -409,6 +421,12 @@ export default function RepresentationEstimator({
   representations,
   streamEvents$,
 } : IRepresentationEstimatorArguments) : Observable<IABREstimate> {
+  const L2A = new L2ARule({
```

Comment:
"Rule" sounds a little too dashjs-y to me and kind of looks out of place here, no ?